src/uct/cuda/gdr_copy/Makefile.am

```diff
@@ -6,13 +6,11 @@
 if HAVE_GDR_COPY
 
 module_LTLIBRARIES              = libuct_cuda_gdrcopy.la
-libuct_cuda_gdrcopy_la_CPPFLAGS = $(BASE_CPPFLAGS) $(CUDA_CPPFLAGS) $(GDR_COPY_CPPFLAGS)
-libuct_cuda_gdrcopy_la_CFLAGS   = $(BASE_CFLAGS)
```

Comment:
why removed libuct_cuda_gdrcopy_la_CFLAGS ?

Reply:
Returned back, because there is no need to remove the flags.